.travis.yml

```diff
@@ -1,7 +1,11 @@
+# Handle git submodules yourself
+git:
+    submodules: false
+# See: https://gist.github.com/iedemam/9830045
+before_install:
+    - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```

Comment:
so using HTTPS here instead of SSH?

Reply:
Can we not just change .gitmodules and be done with it?

Reply:
I do believe it's an issue with CI. I'll try changing .gitmodules and see though!